src/components/DropMenu/Anchor.js

```diff
@@ -4,38 +4,43 @@ import { getThemeProp, ifFlag } from "../../utils";
 import Icon from "../Icon";
 
 export const Header = styled.div`
+	display: flex;
 	cursor: pointer;
 	color: ${ifFlag(
 		"open",
-		getThemeProp(["colors", "application", "base"], "#ccc"),
+		getThemeProp(["colors", "application", "primary"], "#ccc"),
 		"#ccc",
 	)};
 
 	&:hover {
-		color: ${getThemeProp(["colors", "application", "base"], "#ccc")};
+		color: ${getThemeProp(["colors", "application", "primary"], "#ccc")};
 	}
 `;
 
+export const IndicatorWrapper = styled.div``;
+
 export const Indicator = styled(Icon).attrs(props => ({
 	id: ifFlag(
 		"open",
 		getThemeProp(["icons", "indicators", "up"], "chevron-up"),
 		getThemeProp(["icons", "indicators", "down"], "chevron-down"),
 	)(props),
 }))`
-	font-size: 10px;
+	font-size: 12px;
 	padding: 0 11px;
 	color: ${ifFlag(
 		"open",
 		"#ccc",
-		getThemeProp(["colors", "application", "base"], "#ccc"),
+		getThemeProp(["colors", "application", "primary"], "#ccc"),
 	)};
 `;
 
 const Anchor = ({ id, onClick, className, menuLabel, open }) => (
 	<Header {...{ id, onClick, className, open }}>
 		{menuLabel}
-		<Indicator open={open} />
+		<IndicatorWrapper>
+			<Indicator open={open} />
+		</IndicatorWrapper>
```

Comment:
This wrapper serves no apparent purpose - I advise removing it. Never add extraneous elements to the DOM tree.

Reply:
Oh well, I had added for styling reason originally, but now, it looks without it. Removed.